packages/python/plotly/plotly/express/_imshow.py

```diff
@@ -257,10 +312,27 @@ def imshow(
 
     # --------------- Starting from here img is always a numpy array --------
     img = np.asanyarray(img)
+    # Reshape array so that animation dimension comes first, then facets, then images
+    if facet_col is not None:
+        img = np.moveaxis(img, facet_col, 0)
+        if animation_frame is not None and animation_frame < facet_col:
+            animation_frame += 1
+        facet_col = True
+    if animation_frame is not None:
+        img = np.moveaxis(img, animation_frame, 0)
+        animation_frame = True
+        args["animation_frame"] = (
+            "slice" if labels.get("animation") is None else labels["animation"]
```

Comment:
for PX consistency this should be `animation_frame` rather than `slice`/`animation`

Reply:
ok